File: UI/ViewModels/VersionViewModel.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UI.Models;

namespace UI.ViewModels {

    public class VersionViewModel {

        public VersionModel Model { get; set; }

        public VersionViewModel() {
            Model = new VersionModel();
            Model.VersionUrl = @"https://raw.githubusercontent.com/j-alexander/binary-log-search/master/Setup.version";
            Model.BrowseUrl = @"https://github.com/j-alexander/binary-log-search";
            Model.Check();
        }

        public void OnBrowseRequest() {
            Process.Start(Model.BrowseUrl);
        }
    }
}

File: UI/ViewModels/VersionViewModel.fs
namespace UI.ViewModels

open System.Diagnostics

open UI.Commands
open UI.Models

type VersionViewModel() =

    let model = 
        let model = new VersionModel()
        model.VersionUrl <- @"https://raw.githubusercontent.com/j-alexander/binary-log-search/master/Setup.version"
        model.BrowseUrl <- @"https://github.com/j-alexander/binary-log-search"
        model.Check()
        model

    let browseCommand =
        new RelayCommand(
            (fun _ -> true),
            (fun _ -> model.BrowseUrl |> Process.Start |> ignore))

    member public x.Model = model
    member public x.Browse = browseCommand